website/common/locales/en_GB/character.json

```diff
@@ -112,6 +112,7 @@
     "mystery": "Mystery",
     "changeClass": "Change Class, Refund Attribute Points",
     "lvl10ChangeClass": "To change class you must be at least level 10.",
+    "invalidClass":"Invalid class. Please specify warrior, rogue, wizard, or healer.",
```

Comment:
can you revert the changes to the en_GB locales? On github you should only change the American English files, the other languages are managed using Transifex